src/ray/gcs/gcs_server/test/gcs_placement_group_manager_test.cc

```diff
@@ -306,8 +311,15 @@ TEST_F(GcsPlacementGroupManagerTest, TestRescheduleWhenNodeDead) {
   ASSERT_EQ(finished_placement_group_count, 0);
   ASSERT_EQ(mock_placement_group_scheduler_->placement_groups_.size(), 1);
   auto placement_group = mock_placement_group_scheduler_->placement_groups_.back();
+
+  // Manully assign the node field of the first request bundle, simulation scheduling is
+  // done.
```

Comment:
Why do we simulate this in this way? Isn't it enough just calling `OnPlacementGroupCreationSuccess ` or `OnPlacementGroupCreationFailed`?

Reply:
`OnPlacementGroupCreationSuccess` is just a callback, It will not assign node to a placement group request, It's more like simulate the `ScheduleUnplacedBundles` method of `MockPlacementGroupScheduler`